zerver/middleware.py

```diff
@@ -420,7 +420,11 @@ def process_response(
         if requestor_for_logs is None:
             # Note that request.user is a Union[RemoteZulipServer, UserProfile, AnonymousUser],
             # if it is present.
-            if hasattr(request.user, "format_requestor_for_logs"):
+            if remote_server := request_notes.remote_server:
```

Comment:
Is the walrus operator `:=` intended here, or you mean `==` or something?

I think this would be more readable with just a `if request_notes.remote_server is not None` or something.

Reply:
Yes, it was intended. Let's rewrite it then.